### PR TITLE
EPMRPP-76703 || input numeric

### PR DESCRIPTION
**Pull request by chivekrodis**

### CODE REVIEW

**Review by Vadim73i — APPROVED** [review]

File: app/src/componentLibrary/inputNumeric/inputNumeric.jsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            disabled={disabled}
+            min={min}
+            max={max}
+            onKeyDown={disabled ? null : handleKeyDown}
+            onChange={disabled ? null : handleChange}
+            onFocus={disabled ? null : onFocus}
+            onBlur={disabled ? null : onBlur}
+            style={{ width: inputWidth }}
+          />
+          {!!postfix && (value === 0 || !!value) && <span>{postfix.slice(0, 1)}</span>}
```
Comment (AmsterGet) [inline app/src/componentLibrary/inputNumeric/inputNumeric.jsx:123]: What is the reason of doing this `postfix.slice(0, 1)`?

File: app/src/componentLibrary/inputNumeric/inputNumeric.jsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  const handleInputFieldClick = () => {
+    if (inputRef && inputRef.current && inputRef.current.focus) {
+      inputRef.current.focus();
+    }
+    onFocus();
+  };
+
+  return (
+    <div className={cx('container', { disabled })}>
+      {label && <div className={cx('label')}>{label}</div>}
```
Comment (AmsterGet) [inline app/src/componentLibrary/inputNumeric/inputNumeric.jsx:94]: Can we use `span` instead?

Comment (AmsterGet) [inline app/src/componentLibrary/inputNumeric/inputNumeric.jsx:94]: Use `span` instead.

File: app/src/componentLibrary/inputNumeric/inputNumeric.scss [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  font-family: $FONT-ROBOTO-MEDIUM;
+  font-size: 13px;
+  line-height: 20px;
+  text-indent: 1px;
+  color: $COLOR--almost-black;
+}
+
+.sign {
+  width: 16px;
+  height: 16px;
```
Comment (AmsterGet) [inline app/src/componentLibrary/inputNumeric/inputNumeric.scss:37]: Please add `display: inline-block` to be able to use size properties for inline element.

Comment (AmsterGet) [inline app/src/componentLibrary/inputNumeric/inputNumeric.scss:37]: Please add `display: inline-block` to be able to use size properties for inline element.

File: app/src/componentLibrary/inputNumeric/inputNumeric.jsx (hunk 1 above)
Comment (chivekrodis) [reply app/src/componentLibrary/inputNumeric/inputNumeric.jsx:123]: To prevent entering more than one character

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**